### PR TITLE
Fixing GH Actions

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/docbuild.yml [hunk 1]
```diff
@@ -19,4 +22,4 @@ jobs:
       - run: pip install mkdocs-material
       - run: npm install
       - run: npx typedoc --out docs/api lib/index.ts
-      - run: mkdocs gh-deploy
+      - run: mkdocs gh-deploy --force
```
Comment (shapirov103) [inline .github/workflows/docbuild.yml:25]: what is up with --force? I never had to it when deploying from my box.

**Review by shapirov103 — COMMENTED** [review]
Please see my comment. I am a bit unclear on how to test this since we already released, so I really don't want to run another release. I can manually invoke the action, but unsure if it has any dependency on the event data.

File: .github/workflows/docbuild.yml (hunk 1 above)
Comment (elamaran11) [reply .github/workflows/docbuild.yml]: Its not required, i added it to try out to see if any issues. Removed it now.

Comment (elamaran11) [reply .github/workflows/docbuild.yml:25]: It works well only if you have `--force` with GH Actions. See it is tested with this PR and works fine.

**Comment by elamaran11** [comment]
> Please see my comment. I am a bit unclear on how to test this since we already released, so I really don't want to run another release. I can manually invoke the action, but unsure if it has any dependency on the event data.

You can test it by adding your branch on Push like i did it with the PR. If you push a commit to this PR, it will test it.

File: .github/workflows/docbuild.yml [hunk 2]
```diff
@@ -2,6 +2,9 @@ name: 'Release Website Publish'
 on:
   release:
     types: [published]
+  push:
+    branches:
```
Comment (shapirov103) [inline .github/workflows/docbuild.yml:6]: do we still need this?

File: lib/builders/observability-builder.ts [hunk 3]
```diff
@@ -1,24 +1,26 @@
-import * as blueprints from '../../lib';
+import * as blueprintbuilder from '../../lib/stacks';
```
Comment (shapirov103) [inline lib/builders/observability-builder.ts:1]: let's replace with `import { BlueprintBuilder } from ...` and just use it as BlueprintBuilder

File: lib/builders/observability-builder.ts [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
+import * as addons from '../../lib/addons';
 import * as utils from "../utils";
+import * as spi from '../../lib/spi'
```
Comment (shapirov103) [inline lib/builders/observability-builder.ts:4]: `make lint` - I see GH actions missing semicolons and such

**Review by shapirov103 — COMMENTED** [review]
A couple of comments

File: lib/builders/observability-builder.ts (hunk 3 above)
Comment (elamaran11) [reply lib/builders/observability-builder.ts]: fixed

File: lib/builders/observability-builder.ts (hunk 4 above)
Comment (elamaran11) [reply lib/builders/observability-builder.ts]: fixed

**Comment by elamaran11** [comment]
> A couple of comments

@shapirov103 All comments fixed, pls check

**Review by shapirov103 — APPROVED** [review]
LGTM

**Review by shapirov103 — APPROVED** [review]

**Comment by shapirov103** [comment]
@elamaran11 can you confirm that local test provisioning and destroying the cluster was successful?

**Comment by elamaran11** [comment]
> @elamaran11 can you confirm that local test provisioning and destroying the cluster was successful?

@shapirov103 Yes both actions are successful in my local test.

**Pull request merged**